src/scripts/pbs_status.py

```diff
@@ -358,7 +358,7 @@ def get_finished_job_stats(jobid):
                         factor = 1024 * 1024 * 1024
                     elif value[-1] == 'P':
                         factor = 1024 * 1024 * 1024 * 1024
-                        return_dict["ImageSize"] += int(value.strip('KMGTP')) * factor
+                        return_dict["ImageSize"] += int(float(value.strip('KMGTP'))) * factor
```

Comment:
Can this _POSSIBLY_ be right?

Why does `return_dict["ImageSize"]` only get updated when `MaxRSS` ends in the (probably never encountered) `P` suffix?

I can only guess the indentation is wrong.

```suggestion
                    return_dict["ImageSize"] += int(float(value.rstrip('KMGTP'))) * factor
```

Reply:
Argh, good catch! Looks like that bug's been around for a while...